### DISCUSSION TITLE
Misleading tab interface on items in explore feed combined with arbitrary active "tab" selection obfuscates useful information

### DISCUSSION THREAD

**Discussion by milesrichardson** [community/community#27892] [Other Feature Feedback, Questions, & Ideas]
The explore feed is great. By "explore" feed, I mean the one `https://github.com/explore` and also the "topic" feed at `https://github.com/topics/sql`. I frequently use both of these to discover new repositories. 

When I'm searching for a repository, I scroll through the feed and quickly decide which I want to examine further. The first question I need to answer for each repository is "what is this?" For many items in the feed, I can answer this question, because the `<> Code` tab is active, so I can see the readme and repository description.

It's clear what's in this repository:

<img width="903" alt="image" src="https://user-images.githubusercontent.com/835921/182154403-5929c241-9f3d-4936-8ef3-c0db6053d8f2.png">

However, for apparently arbitrary reasons, some items have an active tab other than `<> Code`. I can see how this makes the page less monotonous. But it also obscures information. 

It's not clear what's in this repository:

<img width="864" alt="image" src="https://user-images.githubusercontent.com/835921/182154594-93dc1990-f23f-42fe-b3c8-3448ea2c0ef2.png">

This is where the issue becomes frustrating. I want to see what this repository does. So I click the `<> Code` tab. But it's not a tab! It's a link to the repository. (On mobile, this is extra frustrating, since clicking the link navigates away from the page without restoring pagination when I return).

Can we make this a tab? Please???

<img width="885" alt="image" src="https://user-images.githubusercontent.com/835921/182155395-14955a5b-5d94-4c8e-8bc1-0ae16c890cdf.png">

**Comment by jfullstackdev** [comment]
I tried it, the reason why the Issues tab is displayed is for it to be previewed by other developers that may help the issue to be solved

sometimes it's the Code description that is displayed but when there is a new issue or a very active one, it's being displayed too so developers may find it interesting, 

and it's not a fake tab, if you click on that tab it will go to the repo landing page

**Reply by milesrichardson** [reply]
> I tried it, the reason why the Issues tab is displayed is for it to be previewed by other developers that may help the issue to be solved

I understand that, it makes sense. It's only frustrating because there is no way to open the readme tab instead, without navigating to a new page.

> and it's not a fake tab, if you click on that tab it will go to the repo landing page

Yes, it is a fake tab, in the sense it violates expectations of the average user. In 99% of web interfaces, a Tab is an element that you click to change the local context of the tab group. Sometimes, this might be implemented with a Link, but the destination is still the same page but a different active tab. Whereas in this case, the Link goes to _an entirely different page_. This is not what a user expects to happen when clicking a Tab element.

Here is a [component gallery of hundreds of tab implementations](https://component.gallery/components/tabs/). None of them work like the fake tabs in the feed here. That page also includes a description of a Tab component:

> A tabbed interface consists of two key parts:
> 
> - the tab list containing the tab elements, along the top or side of…
> - the tab panels which contain the content.
>
> Each tab panel has an associated tab in the tab list, that when activated, displays the panel. Only one panel of content is shown at any one time.
>
> Tabs are an example of an [interface metaphor](https://en.wikipedia.org/wiki/Interface_metaphor), named after the tabs attached to the top of folders used inside filing cabinets. Each tab is given a label based on the documents inside its corresponding folder. Like their real-world namesake tabbed interfaces allow a user to see the outline of a larger set of data at a glance and quickly navigate to the desired section.

This is not the case here. Fake tabs.

**Reply by jfullstackdev** [reply]
> but the destination is still the same page but a different active tab. Whereas in this case, the Link goes to _an entirely different page_. This is not what a user expects to happen when clicking a Tab element.

it goes exactly to that page, 

![image](https://user-images.githubusercontent.com/47092464/182666337-754cc3d7-2786-409b-8a4f-9e91d7e80089.png)

if I click `Code`, this is the page,

![image](https://user-images.githubusercontent.com/47092464/182666390-67247d08-b3b8-4724-bde7-bee115d98d77.png)

if I click `Pull Requests`, this is the page,

![image](https://user-images.githubusercontent.com/47092464/182666437-6a9e4fa2-578a-48f2-a839-8d3d9d80458d.png)

**Reply by milesrichardson** [reply]
It navigates away from the page with the feed.

**Reply by pcjmfranken** [reply]
> other developers that may help the issue to be solved

How this this supposed to work, though, without even knowing what language the project is written in or any clue as to what the repository is about (besides the topic)? This information is all hidden when this useless issues tab is displayed.

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
